index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" integrity="sha512-iBBXm8fW90+nuLcSKlbmrPcLa0OT92xO1BIsZ+ywDWZCvqsWgccV3gFoRBv0z+8dLJgyAHIhR35VZc2oM/gI1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="text-center">Leaderboard</h1>
+    <div class="wrapper">
+        <section class="show-score">
+            <div class="header-score mb-2">
+                <h2>Recent score</h2>
+                <button class="refresh"> Refresh <i class="fas fa-sync-alt"></i> </button>
+            </div>
+            <div id="user-table">
+
+            </div>
+        </section>
+        <section class="add-score">
+            <h2>Add your score</h2>
+            <form class="form">
+                <input type="text" placeholder="Your name">
+                <input type="text" placeholder="Your score">
+                <input type="submit">
+            </form>
+        </section>
+    </div>
```

Comment:
- [ ] KIndly let's move all this code to `index.js`. Index.html should only contain the root element which in this case is `index.js` Don't forget to apply ES6 best practises such as `arrow functions` instead of `function` keyword etc 😄